Assets/RGLUnityPlugin/Scripts/LidarSensor.cs

```diff
@@ -220,11 +233,27 @@ public void Capture()
         {
             sceneManager.DoUpdate();
 
+            //Calculate delta transform of lidar.
+            Matrix4x4 currentTransform = gameObject.transform.localToWorldMatrix * configuration.GetLidarOriginTransfrom();
+            Vector3 deltaTranslation = (lastTransform.GetColumn(3) - (currentTransform).GetColumn(3)) * 0.001f;
+
+            Vector3 deltaRotation = Quaternion.LookRotation(lastTransform.GetColumn(2), lastTransform.GetColumn(1)).eulerAngles
+                                    - Quaternion.LookRotation(currentTransform.GetColumn(2), currentTransform.GetColumn(1)).eulerAngles;
+
+            deltaRotation *= Mathf.Deg2Rad;
```

Comment:
I'd warp it into the function and calculate it only when `applyVelocityDistortion` is true.